test/addons-napi/test_properties/test_properties.c

```diff
@@ -37,6 +37,30 @@ napi_value Echo(napi_env env, napi_callback_info info) {
   return args[0];
 }
 
+napi_value HasNamedProperty(napi_env env, napi_callback_info info) {
+  size_t argc = 2;
+  napi_value args[2];
+  NAPI_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  NAPI_ASSERT(env, argc == 2, "Wrong number of arguments");
+
+  // Extract the name of the property to check
+  char buffer[128];
+  size_t buffer_size = 128;
+  size_t copied;
+  buffer[buffer_size - 1] = 0;
```

Comment:
`napi_get_value_string_utf8()` will always null-terminate, so this line should not be necessary.